src/figuresClasses.ts

```diff
@@ -1,19 +1,97 @@
-export interface Figure {
+const ERROR_MESSAGE = 'Value cannot be <= 0!';
```

Comment:
👍 
But try to make up a more descriptive name, which type of error message it is